README.md

```diff
@@ -387,6 +388,40 @@ E.g. [Docker Hub](https://hub.docker.com/), [Quay.io](https://quay.io/repository
    finishes successfully, and that your public repository contains the images pushed by
    the pipeline.
 
+
+#### Automatically Provision Quay Repositories For Container Images
+
+**Note**: This step is mandatory for importing components using the UI.
+
+Konflux integrates with the [Image Controller](https://github.com/konflux-ci/image-controller)
+that can automatically create Quay repositories when onboarding a component.
+
+The controller requires a token for a Quay application that has the following permissions on an
+organization:
+
+For creating a Quay application please follow
+
+1. [Create a user on Quay.io](https://quay.io/)
+
+2. [Create Quay Organization](https://docs.projectquay.io/use_quay.html#org-create)
+
+3. [Create Quay Application](https://docs.projectquay.io/use_quay.html#_create_oauth_access_token) - The application should have the following permissions
+
+   3.1 Administer Organization
+   3.2 Administer Repositories
+   3.3 Create Repositories
+
+4. Run the `deploy-image-controller.sh` script:
+
+`$TOKEN` - the token for the application you've created on step 3.
+
+`$Orgnization` - the name of the organization you created on step 2.
```

Comment:
There's a typo on the word, and it should be capitalized.
```suggestion
`$ORGANIZATION` - the name of the organization you created on step 2.
```

Reply:
donee